modules/data.remote/R/call_MODIS.R

```diff
@@ -13,7 +13,10 @@
 ##' @param band   string value for which measurement to extract
 ##' @param band_qc string value for which quality control band, or use "NA" if you do not know or do not need QC information (optional)
 ##' @param band_sd string value for which standard deviation band, or use "NA" if you do not know or do not need StdDev information (optional)
+##' @param siteID numeric value of BETY site id value to use for output file name. Default is NULL
 ##' @param package_method string value to inform function of which package method to use to download modis data. Either "MODISTools" or "reticulate" (optional)
+##' @param QC_filter Converts QC values of band and keeps only data values that are excellent or good (as described by MODIS documentation), and removes all bad values. qc_band must be supplied for this parameter to work. Default is False.
+##' @param progress TRUE reports the download progress bar of the dataset, FALSE omits the download progress bar. Default is TRUE.
```

Comment:
This is the parallel version of progress we discussed a month or so ago correct?

Reply:
I tried parallel, it doesn't make the function go any faster because right now the function is set up to go by a site, vs. multiple sites. In the future I'll probably change it so that you can give it a list of sites or lat/lons and then I can parallelize it easily.